easybuild/easyconfigs/k/Kraken2/Kraken2-2.1.4-gompi-2024a.eb

```
@@ -0,0 +1,75 @@
+# Update to version 2.1.4 means the tarballs is version 2.14.tar.gz!
+# Hence all that string substitution. Hopefully that will be changed
+# back with a newer update.
+# Author: J. Saßmannshausen (Imperial College London/UK) + Alexander Grund
+
+easyblock = 'PackedBinary'
+
+name = 'Kraken2'
+version = '2.1.4'
+
+homepage = 'https://github.com/DerrickWood/kraken2/wiki'
+description = """Kraken is a system for assigning taxonomic labels to short DNA sequences,
+ usually obtained through metagenomic studies. Previous attempts by other
+ bioinformatics software to accomplish this task have often used sequence
+ alignment or machine learning techniques that were quite slow, leading to
+ the development of less sensitive but much faster abundance estimation
+ programs. Kraken aims to achieve high sensitivity and high speed by
+ utilizing exact alignments of k-mers and a novel classification algorithm."""
+
+# part is compiled with $CXX, the rest is in Perl
+toolchain = {'name': 'gompi', 'version': '2024a'}
+toolchainopts = {'openmp': True, 'cstd': 'c++11'}
+
+github_account = 'DerrickWood'
+source_urls = [GITHUB_LOWER_SOURCE]
+# The version is 2.1.4 but the tarball is 2.14, hence we need to do this:
+sources = ['v%s.tar.gz' % version.replace('1.4', '14')]
```

this clearly is a mistake, so let's make sure the replacement is only done for the exact version?
```suggestion
sources = [f'v{version.replace("2.1.4", "2.14")}.tar.gz']
```